src/admin.c

```diff
@@ -745,6 +745,34 @@ void AdminForceBreak()
 	EndMatch(0);
 }
 
+void AdminForceMap()
+{
+	if (!is_adm(self))
+	{
+		return;
+	}
+	if (!k_matchLess && match_in_progress)
+	{
+		G_sprint(self, 2, "Match currently in progress. Use %s or %s to terminate.\n", redtext("break"), redtext("forcebreak"));
+		return;
+	}
+	if (trap_CmdArgc() < 2)
+	{
+		G_sprint(self, 2, "Usage: forcemap %s\n", redtext("<mapname>"));
+		return;
+	}
+	char mapname[128];
+	trap_CmdArgv(1, mapname, sizeof(mapname));
+	extern int GetMapNum(char *);
```

Comment:
add the function declaration of GetMapNum() to g_local.h instead. There are already functions from maps.c there.